appgate/client.py

```diff
@@ -238,12 +243,42 @@ async def init(self) -> None:
             self.namespace,
             self.name,
         )
+
+        def configmap_exists() -> bool:
+            configmaps = self._v1.list_namespaced_config_map(namespace=self.namespace)  # type: ignore
+            for configmap in configmaps.items:
+                if configmap.metadata.name == self.name:
+                    return True
+            return False
+
+        def initialize_configmap() -> V1ConfigMap:
+            if configmap_exists():
```

Comment:
I would do some `get_configmap` returning `Optional`, then you just check if it's None or not